### PR TITLE
[EFR05] Enterprise Feature Request: AAR and JAR support

### PR DESCRIPTION
**Pull request by ajinabraham**
### Describe the Pull Request

```
Support AAR and JAR file scanning and report generation.
```

### Checklist for PR

- [x] Run MobSF unit tests and lint `tox -e lint,test`
- [x] Tested Working on Linux, Mac, Windows, and Docker
- [x] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [x] Make sure tests are passing on your PR [MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

**Review by superpoussin22 — APPROVED** [review]
sounds good on my side , it works with 2 aar I upload to MobSF

File: mobsf/StaticAnalyzer/views/android/generate_downloads.py [hunk 1]
```diff
@@ -15,39 +15,43 @@
...
+        elif file_type in ('apk', 'ipa', 'jar', 'aar'):
+            # Binaries
             file_name = f'{md5}.{file_type}'
-            src = os.path.join(app_dir, file_name)
-            dst = os.path.join(settings.DWD_DIR, file_name)
-            shutil.copy2(src, dst)
-        return redirect('/download/' + file_name)
+            src = app_dir / file_name
+            dst = dwd_dir / file_name
+            shutil.copy2(src.as_posix(), dst.as_posix())
```
[the remark this answers is not in the document]
Comment (ajinabraham) [reply mobsf/StaticAnalyzer/views/android/generate_downloads.py:52]: False Positive

[the remark this answers is not in the document]
Comment (ajinabraham) [reply mobsf/StaticAnalyzer/views/android/generate_downloads.py:52]: False Positive

File: mobsf/StaticAnalyzer/views/android/generate_downloads.py [hunk 2]
```diff
@@ -15,39 +15,43 @@
...
+        return redirect(f'/download/{file_name}')
```
[the remark this answers is not in the document]
Comment (ajinabraham) [reply mobsf/StaticAnalyzer/views/android/generate_downloads.py:53]: False Positive

**Pull request merged**